docs/src/modules/ROOT/pages/using-timefold-solver/benchmarking-and-tweaking.adoc

```diff
@@ -753,6 +774,34 @@ After those few seconds of initialization, the calculation speed is relatively s
 ====
 
 
+[#benchmarkReportMoveEvaluationSpeedOverTimeStatistic]
+=== Move evaluation speed over time statistic (graph and CSV)
+
+To see how fast the moves are evaluated, add:
+
+[source,xml,options="nowrap"]
+----
+    <problemBenchmarks>
+      ...
+      <problemStatisticType>MOVE_EVALUATION_SPEED</problemStatisticType>
+    </problemBenchmarks>
+----
+
+.Move Evaluation Speed Statistic
+image::using-timefold-solver/benchmarking-and-tweaking/moveEvaluationSpeedStatistic.png[align="center"]
+
+
+[NOTE]
+====
+The initial high calculation speed is typical during solution initialization:
+it's far easier to calculate the score of a solution if only a handful planning entities have been initialized,
+than when all the planning entities are initialized.
+
+After those few seconds of initialization, the evaluation speed is relatively stable,
```

Comment:
```suggestion
After the construction heuristic phase, the evaluation speed is relatively stable,
```

Let's not promise seconds. :-) We've seen it can be minutes, hours.